packages/deprecated/src/index.js

```diff
@@ -30,6 +43,20 @@ export default function deprecated( feature, { version, alternative, plugin, lin
 		return;
 	}
 
+	/**
+	 * Fires whenever a deprecated feature is encountered
+	 *
+	 * @param {string}  feature             Name of the deprecated feature.
+	 * @param {?Object} options             Personalisation options
+	 * @param {?string} options.version     Version in which the feature will be removed.
+	 * @param {?string} options.alternative Feature to use instead
+	 * @param {?string} options.plugin      Plugin name if it's a plugin feature
+	 * @param {?string} options.link        Link to documentation
+	 * @param {?string} options.hint        Additional message to help transition away from the deprecated feature.
+	 * @param {?string} message				Message sent to console.warn
```

Comment:
In the last line of the comment, there is some spacing issue. I think there are used tabs instead of spaces.

Reply:
fixed. thanks